src/index.html

```diff
@@ -210,95 +254,163 @@
-            <input id="iiifManifestId" type="text" value="" class="flex-1" />
-          </div>
-          <div class="mt-2">
-            <button
-              id="setIIIFManifestIdButton"
-              class="controls-button"
-              href="#"
-            >
-              Set IIIF Manifest Id
-            </button>
-          </div>
-        </div>
+     <div id="iiifTab" class="controls-tab-content">
+      <div class="tab">
+        <button class="tablinks active" onclick="openTab(event, 'general')">General</button>
+        <button class="tablinks" onclick="openTab(event, 'osd')">OpenSeadragon</button>
+        <button class="tablinks" onclick="openTab(event, 'av')">AV</button>
+        <button class="tablinks" onclick="openTab(event, 'annotations')">Annotations</button>
+        <button class="tablinks" onclick="openTab(event, 'config')">Configuration</button>
+      </div>
 
-        <div class="controls-group">
-          <h2 class="controls-title">Target</h2>
-          <div class="flex">
-            <input id="iiif_target" type="text" value="" class="flex-1" />
-            <button
-              id="iiif_setTargetButton"
-              class="controls-button flex-initial ml-2"
-            >
-              Set Target
-            </button>
-          </div>
-        </div>
+    <!-- General -->
+    <div id="general" class="tabcontent">
+      <h2 class="controls-title">Manifest Id</h2>
+      <div class="flex">
+        <select id="iiifManifestIdSelect" class="py-2 px-3 mr-2 flex-1"></select>
+        <input id="iiifManifestId" type="text" value="" class="flex-1" />
+      </div>
+      <div class="mt-2">
+        <button id="setIIIFManifestIdButton" class="controls-button">Set IIIF Manifest ID</button>
+      </div>
+    <div class="controls-group">
+      <h2 class="controls-title">Set Target</h2>
+      <div class="flex mb-2">
+        <input id="iiif_target" type="text" value="" class="flex-1" />
+        <button id="iiif_setTargetButton" class="controls-button flex-initial ml-2">
+          Set Target
+        </button>
+      </div>
+    <div class="mt-2">
+      For OpenSeadragon manifests, edit the 'xywh' values at the end of the URL to modify the view of the image.
+      For AV manifests, modify the number after 't' to set the timestamp.
+    </div>
+  </div>
+  </div>
+
+    <!-- OSD -->
+    <div id="osd" class="tabcontent">
+    <div class="controls-group">
+      <h2 class="controls-title">Set degrees of rotation</h2>
+      <div class="flex">
+        <input id="iiif_rotation" type="range" value="0" min="-180" max="180" class="mr-2" />
+        <span id="iiif_rotationValue" class="mr-2">0</span>
+        <button id="iiif_setRotationButton" class="controls-button flex-initial ml-2">
+          Set Rotation
+        </button>
+        <button id="iiif_resetRotationButton" class="controls-button flex-initial ml-2" style="padding: 0.5rem 0.5rem; font-size: 1rem; min-width: 0;">
+        Reset
+      </button>

```

Comment:
Indentation issue here as well:

```suggestion
        <button id="iiif_resetRotationButton" class="controls-button flex-initial ml-2" style="padding: 0.5rem 0.5rem; font-size: 1rem; min-width: 0;">
          Reset
        </button>
```

Reply:
It might also make sense to give this button a `display: none` style for now with a TODO comment to fix it since it doesn't work right yet; we can reactivate it when we fix the underlying rotation issues, which seems best addressed in a follow-up PR.